src/components/Navigation.jsx

```diff
@@ -1,4 +1,68 @@
+"use client";
+import Logo from "@/public/images/WITL-logo.webp";
+import Image from "next/image";
+import Link from "next/link";
+import React, { useState } from "react";
+import { items } from "@/data/Nav.js";
+import { FaBars } from "react-icons/fa";
+
 const Navigation = () => {
-  return <div></div>;
+  const [mobile, setMobile] = useState(false);
+  const handleMobileMenu = () => {
+    setMobile(!mobile);
+  };
+  return (
+    <div className="sticky top-0">
+      <div className="w-screen bg-witl-purple-300 flex items-center justify-between py-[1%] px-[2%] border-b-2 border-witl-white-100">
+        <Link className="flex flex-row space-x-4" href="/">
+          <Image src={Logo} alt="WITL Icon" />
+          <div className="hidden md:flex md:flex-col text-witl-white-100">
+            <div className="text-2xl">WOMXN IN THE LAW</div>
+            <div>EST. 2022</div>
+          </div>
+        </Link>
+
+        <div className="md:hidden flex flex-col items-center text-witl-white-100">
+          <div className="text-xl">WOMXN IN THE LAW</div>
+          <div>EST. 2022</div>
+        </div>
+
+        <div className="hidden md:flex space-x-16">
```

Comment:
space-x-8 probably?? and the distance will match the figma perfectly and that's all